treeherder/seta/management/commands/reset_seta.py

```diff
@@ -1,13 +1,18 @@
 from django.core.management.base import BaseCommand
```

Comment:
That's not what I'm saying. We already load the preseed file every time (and should).
It's just this PR from what I can see deletes state from the DB that will have been generated by the last daily run.